schemas/context/profile-privacy.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+		"Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+		"This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+		"you may not use this file except in compliance with the License. You may obtain a copy",
+		"of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-privacy",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Privacy",
+  "description": "Accepts Privacy Opt-Out signals for Selling/Sharing of data or general Opt-Outs by Users",
```

Comment:
This description needs to be reworded and properly captialized.

Reply:
Mixin is an internal word. Should we refrain from using Mixin in the descriptions ?

Reply:
Should it be "Profile privacy schema to capture general and sales/sharing opt-out signals."

Reply:
Replacing `mixin` with `schema` makes sense in this context.